cookies/abtest_test.go

```diff
@@ -0,0 +1,76 @@
+package cookies
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUnitABTest(t *testing.T) {
+
+	var testDomain = "www.ons.gov.uk"
```

Comment:
Can just use `:=` to initialise variable instead of using `var`

Reply:
👍